### PR TITLE
DOCS-604 - Test Wistia video embedding

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request tests embedding a video from Wistia into this article: https://help.sumologic.com/docs/cse/administration/mitre-coverage/

## Select the type of change

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[DOCS-604](https://sumologic.atlassian.net/browse/DOCS-604)

### CODE REVIEW

File: docs/cse/administration/mitre-coverage.md [hunk 1]
```diff
@@ -34,6 +35,9 @@ Watch this micro lesson to learn about the MITRE ATT&CK Threat Coverage Explorer
      allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
      allowfullscreen
      />
+-->
+
+<script src="https://fast.wistia.net/player.js" async></script><iframe src="https://fast.wistia.net/embed/iframe/yebz0v90tx?web_component=true&seo=true&videoFoam=false" title="Micro Lesson: Cloud SIEM MITRE ATT&amp;CK® Threat Coverage Explorer Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" width="854px" height="480px"></iframe>
```
Comment (kimsauce) [inline docs/cse/administration/mitre-coverage.md:40]: Could you change to this format? This allows the video embedding to be responsive in desktop/mobile when users resize the window.

```suggestion
<Iframe url="https://fast.wistia.net/embed/iframe/yebz0v90tx?web_component=true&seo=true&videoFoam=false"
  width="854px"
  height="480px"
  id="wistiaVideo"
  className="video-container"
  display="initial"
  position="relative"
  allow="autoplay; fullscreen"
  allowfullscreen
/>
```

Here's what it looks like, not-responsive code vs responsive code.

https://github.com/user-attachments/assets/14576fa7-9560-4582-b3e2-eae2cfe95452

https://github.com/user-attachments/assets/aa8c08a6-1231-4279-8704-17c03857a64a

**Review by kimsauce — CHANGES_REQUESTED** [review]
change to code

**Comment by jpipkin1** [comment]
Thanks, @kimsauce! Your new code works great.

**Comment by jpipkin1** [comment]
@kimsauce, could you take another look?

**Review by kimsauce — APPROVED** [review]
Perfect!

**Pull request merged**